tests/testthat/test-extract_hash_from_object.R

```diff
@@ -0,0 +1,65 @@
+test_that("extract_hash_from_object string objects work", {
```

Comment:
Would you mind moving these tests into `test-capture-requests.R`? That file isn't too long and I try to keep the test files to match the corresponding R files for ease of single-file testing.